src/libraries/System.Linq.Expressions/src/System.Linq.Expressions.csproj

```diff
@@ -1,9 +1,10 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <FeatureInterpret>true</FeatureInterpret>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-iOS</TargetFrameworks>
```

Comment:
What about other similar targets? tvOS, macCatalyst, WASM?

Reply:
Mac Catalyst is not necessary since it can fall back to iOS. tvOS, however, is necessary. Not quite sure about WASM and which implementation is more suitable. WASM doesn't have the FullAOT limitations at very least.

Reply:
WASM AOT operates exactly like iOS AOT (fallbacks to the interpreter) so I think it makes sense to use the same logic.

Reply:
That doesn't sound the same to me. Wasn't the original issue with the fact that iOS in certain cases doesn't ship the interpreter? Anyway, I have no objection to changing it for WASM too. Just wanted to clarify whether there is the same underlying reason for it.

Reply:
Added iOS-like targets first. If all is good with them, I'll add WASM.

Reply:
If the changes look good, I'd prefer to merge this PR and open a follow-up for WASM.

Reply:
Yeah it looks good to me as is just to make things work. Whether we want to eventually switch back to using FEATURE_COMPILE and running the emitted IL in the Mono interpreter on various platforms is a separate question.